stubs/usb/core.pyi

```diff
@@ -4,10 +4,13 @@ Type stubs for usb.core
 Note that stubs are only written for the parts that we use.
 """
 
-from typing import Generator, Optional, Union
+from typing import Generator, Optional, Tuple, Union
 
 
 class Device:
+
+    _langids: Tuple[int, ...]
```

Comment:
Is this used?

Reply:
Ah this was something I was using for debugging https://github.com/j5api/sr-servo-v4-fw/issues/5

Shouldn't be here.